docs/UserGuide.adoc

```diff
@@ -172,35 +172,83 @@ Lists all the commands that you have entered in reverse chronological order. +
 Format: `history` +
 Alias: `hist`
 
-=== Creating a new journal entry : `new`
+[NOTE]
+====
+Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
+====
+
+=== Viewing a timetable : `tview`
+
+Shows the current saved timetable. +
+Format: `tview` +
+Alias: `tv`
+
+=== Adding a timetable : `tadd`
+
+Imports a timetable from NUSMODS website. +
+Format: `tadd` +
+Alias: `ta`
```

Comment:
Are we planning to add in the command format now or later?

Format:  `tadd [u/NUSMODS_SHORT_URL]`
Alias:  `ta [u/NUSMODS_SHORT_URL]`

Example:
- `tadd u/http://modsn.us/wNuIW`
- `ta u/http://modsn.us/wNuIW`

Reply:
I think we can add it as we implement our features. Since the skeleton is there already, we will be changing different lines of code so wont have merge conflict

Reply:
Samuel say add concurrently